src/webkit/wkExecutionContext.ts

```diff
@@ -98,13 +102,10 @@ export class WKExecutionContext implements js.ExecutionContextDelegate {
         returnByValue: true
       });
       if (serializeResponse.wasThrown)
-        return undefined;
+        throw new Error('Evaluation failed: ' + serializeResponse.result.description);
```

Comment:
This will likely break some of the code where no exception is currently expected. My understanding was that the current contract of returning undefined instead of throwing was intentional since ling ago. I personally agree that it's not good to silently swallow errors but Lusha/Joel might remember if there was a specific request for that behavior.

Reply:
This current behavior is webkit-specific. It has to do with returnByValue being a separate step from the evaluate itself, due to special promise handling.